test/scripts/test_private_network.sh

```diff
@@ -2,7 +2,10 @@
 echo "######################################################################"
 echo "  test_private_network"
```

Comment:
Somehow I've never noticed these tests. I know `e2e_basic_start_stop.sh` runs. Does this one run in the current CI?

Reply:
e2e_subs are different, they all reuse the same private network

Reply:
I modified the test so I could add it to `e2e.sh`, but realized that it runs serially and adds 20s to the overall test (whereas `e2e_subs` runs more in parallel)